pkg/systemd/terminal.css

```
@@ -16,19 +16,19 @@ html, body {
 }
 
 .black-theme {
-    background-color: #000000;
+    background-color: var(--color-text);
 }
 
 .dark-theme {
-    background-color: #002b36;
+    background-color: var(--color-dark-blue);
```

Revert; this is a standard theme.

```suggestion
    background-color: #002b36;
```